docs/resources/overview.md

```diff
@@ -4,7 +4,7 @@ title: Overview
 ---
 
 You can upload resources in casdoor. Before upload resources, you need to configure a storage provider.
-Please see [Storage Provider](/docs/provider/storage/storage)
+Please see [Storage Provider](/docs/provider/storage)
```

Comment:
refer to: https://docusaurus.io/docs/next/sidebar/autogenerated#category-index-convention

Reply:
good